gui/src/hooks/imu-logic.ts

```diff
@@ -1,11 +1,21 @@
 import { useMemo } from 'react';
 import { FlatDeviceTracker } from './app';
 
+const IGNORED_BOARDS = new Set(['Sony Mocopi', 'Haritora']);
+
 export function useIsRestCalibrationTrackers(
   connectedTrackers: FlatDeviceTracker[]
 ): boolean {
   const imuExists = useMemo(
-    () => connectedTrackers.some((tracker) => tracker.tracker.info?.isImu),
+    () =>
+      connectedTrackers.some(
+        (tracker) =>
+          tracker.tracker.info?.isImu &&
+          !(
+            tracker.device?.hardwareInfo?.boardType &&
+            IGNORED_BOARDS.has(tracker.device?.hardwareInfo?.boardType as string)
```

Comment:
I would suggest using `McuType` but it doesn't look like we have one for Haritora, so whatever

Reply:
but its wrong to call haritora an mcutype, its a DA14